developer/openshift/reset.sh

```diff
@@ -149,21 +179,34 @@ uinstall_minio() {
 uninstall_pipeline_service() {
     printf "\n  Uninstalling Pipeline Service:\n"
     # Remove pipeline-service Argo CD application
-    if ! argocd app get pipeline-service >/dev/null 2>&1; then
-      printf "\n[ERROR] Couldn't find the 'pipeline-service' application in argocd apps.\n" >&2
-      exit 1
+    if [ "$(is_hard_reset_enabled)" != "true" ]; then
+      if ! argocd app get pipeline-service >/dev/null 2>&1; then
+        printf "\n[ERROR] Couldn't find the 'pipeline-service' application in argocd apps.\n" >&2
+        exit 1
+      fi
     fi
 
+    # If something went wrong(f.e. bad development changes) argocd sync operation can be very long or could hang.
+    # In this case all next argocd operations will be delayed.
```

Comment:
`case, the other argocd operations will be queued, which will prevent the deletion of the application.`